### PR TITLE
Update description of vocabulary additions (#658).

### LINKED ISSUE #658: Check changes document and new vocabulary appendix.
Ensure that changes document and new vocabulary appendix are accurate and adequately complete.

### PR DESCRIPTION
**Pull request by skynavga**
Closes #658.

### CODE REVIEW

File: spec/ttml2.xml [hunk 1]
```diff
@@ -21650,7 +21657,7 @@ is optional (O), for transformation and <loc href="#terms-presentation-processor
 <col css="width: 15%;"/>
 <tbody>
 <tr>
-<td><phrase role="strong">Feature</phrase></td>
+<td><phrase role="strong">Feature Designation</phrase></td>
```
Comment (nigelmegitt) [inline spec/ttml2.xml]: Same change needs to be made to tables E-2 and E-3 for consistency.

Comment (skynavga) [reply spec/ttml2.xml]: Fixed, along with a table formatting issue in E-1 for new ``#extent-*`` designators.

File: spec/.#ttml2-changes.html [hunk 2]
```diff
@@ -0,0 +1 @@
+glenn@sn-ga-03.local.918
```
Comment (nigelmegitt) [inline spec/.#ttml2-changes.html]: Should this be in here?

File: spec/ttml2-changes.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+d<!DOCTYPE html>
```
Comment (nigelmegitt) [inline spec/ttml2-changes.html]: typo?

File: spec/.#ttml2-changes.html (hunk 2 above)
Comment (skynavga) [reply spec/.#ttml2-changes.html]: @nigelmegitt I don't see this in my working copy (now)

File: spec/ttml2-changes.html (hunk 3 above)
Comment (skynavga) [reply spec/ttml2-changes.html]: Fixed.

**Pull request merged**